subsys/net/lib/nrf_cloud/src/nrf_cloud_codec_internal.c

```diff
@@ -1391,7 +1395,7 @@ static int encode_modem_info_device(struct device_param *device, cJSON *json_obj
 		return -ENOMEM;
 	}
 
-	if (json_add_str_cs(json_obj, "sdkVer", device->app_version)) {
```

Comment:
this "app_version" comes from modem_info lib.
it holds the SDK version... so perhaps it should be renamed in modem_info?

example:
<img width="594" alt="image" src="https://github.com/nrfconnect/sdk-nrf/assets/4138391/4384bff5-5521-484b-bb34-55d0230118d7">

do you see different behavior?

Reply:
Sorry for the late reply. In the [OOB firmware](https://github.com/hello-nrfcloud/firmware), the sdkVer appears as APP_VERSION. This might be because it's out-of-tree, where the CMAKE variable isn't available, but I'm not certain.

I didn't see why a modem library should be used to retrieve the SDK version, so I found it more appropriate to fix the issue by referencing the actual NCS version and commit strings, which should always be available.

The modem info library parameter refers to an APP_VERSION, which isn't really an SDK version at all. This is another reason not to use it.

Overall, I don't see why the modem info library should provide either the app or SDK version, so I would suggest removing the parameter in a later effort.

Reply:
created `IRIS-9301`